packages/sentry/src/index.ts

```diff
@@ -1,6 +1,10 @@
 export { default as version } from './version';
 export * from './sentry';
-export { default as SentryUserTracker } from './sentry-user-tracker';
+export {
+  default as SentryUserTracker,
+  updateUser,
+} from './sentry-user-tracker';
```

Comment:
The changes here are so that we can define the user type in the react component file, thus getting the generated prop types.

<img width="453" alt="image" src="https://user-images.githubusercontent.com/1110551/59391224-faccec00-8d73-11e9-8565-a5c2bd110380.png">